components/variations/client_filterable_state.cc

```diff
@@ -26,6 +26,11 @@ Study::Platform ClientFilterableState::GetCurrentPlatform() {
   // Default BSD and SOLARIS to Linux to not break those builds, although these
   // platforms are not officially supported by Chrome.
   return Study::PLATFORM_LINUX;
+#elif defined(STARBOARD)
+  // On Cobalt, the current OS/Platform are logged elsewhere. The return value
+  // here doesn't better, but it needs to be something to keep the compiler
+  // happy.
+  return Study::PLATFORM_LINUX;
```

Comment:
Nit: not necessarily wrong, but i'd consider defining `Study::PLATFORM_STARBOARD` for preventing surprises later on.

Reply:
Can do, except it would require adding a STARBOARD entry to study.proto Platform enum. Is that better than this?

Reply:
I think so, as someone using this fn later on for something real would be surprised if they always got the linux value

Reply:
One hang-up here though... if I add an entry to Platform, it could introduce a backwards-incompatible change if Chromium modifies the proto upstream. E.g., the enum is currently:

```
 enum Platform {
    PLATFORM_WINDOWS         = 0;
    PLATFORM_MAC             = 1;
    PLATFORM_LINUX           = 2;
    PLATFORM_CHROMEOS        = 3;
    PLATFORM_ANDROID         = 4;
    PLATFORM_IOS             = 5;
    PLATFORM_ANDROID_WEBVIEW = 6;
    PLATFORM_FUCHSIA         = 7;
  }
```

If I add:

```
  PLATFORM_STARBOARD = 8;
``

We've now effectively reserved field 8. If that ever gets overwritten upstream, it'd be a mess. 

I could either A) leave this change as-is or B) add a purposely very high enum value like `PLATFORM_STARBOARD = 1000`. Thoughts?

Reply:
Can you just add `PLATFORM_STARBOARD = -1` at the top?